### PR TITLE
Updates to Adapters Help Pages

### PR DESCRIPTION
**Pull request by capricornunicorn123**
### Description of the Change
This PR adds help pages to Adapter Data Access Plugins.
Specifically, Gaffer and GDELT related plugins have been created. 

### Benefits
Clarifies the proper use of Adapter Data Access Plugins.

### CODE REVIEW

File: AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/extend-from-gdelt.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Extend From GDELT Knowledge Graph
+
+This plugin extends an existing graph by importing relationships and related entities from the <a href="https://www.gdeltproject.org/">Global Database of Events, Language, and Tone</a>.
+<div style="text-align: center">
+    <img src="../ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/resources/GDELTExtension.png" width="80%" alt="GDELT Extension Example"/>
+</div>
+
+Only relationships involving existing and selected nodes on the graph will be imported. Any Entities not currently on the grap will be added.
```
Comment (Delphinus8821) [inline AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/extend-from-gdelt.md:8]: typo graph

File: AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/extend-from-gdelt.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Entities and relationships are imported based on their position in the incoming data. Elements that are received first will be included in the import. 
+
+To understand moe about GDELT Entities and Relationships in Constellation, see
```
Comment (Delphinus8821) [inline AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/extend-from-gdelt.md:11]: typo more

File: AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/import-entities-from-gdelt.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+-   Identifier: *Location's Extended Name*
+-   Geo.Country: *Country of the Location*
+-   Geo.Longitude: *Longitudinal Coordinates*
+-   Geo.Latitude: *Latitudinal Coordinates*
+
+**Source** - A general point of origin that entity information has been extracted from. Typically, an online publication, forum or site. Contains the attribute:
+
+-   Identifier: *Source's Name*
+
+**URL** - A specific point of origin that entity information has been extracted from. Typically, an online publication, forum or site Typically. Contains the attribute:
```
Comment (Delphinus8821) [inline AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/import-entities-from-gdelt.md:34]: Grammar is a bit weird in this sentence

File: AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/import-entities-from-gdelt.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+A range of different entity types can be imported using this plugin. These entity types are described as follows. 
+
+**Person** - An entity that represents a single person. Typically an individual that may be real or fictional. Contains the attribute:
+
+-   Identifier: *Person's Full Name*
+
+**Organisation** - An entity that represents a collection of people. Typically, a business, government, or charity. Contains the attribute:
+
+-   Identifier: *Organisation’s Name*
```
Comment (Delphinus8821) [inline AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/import-entities-from-gdelt.md:17]: Something weird happens with the ' on this line when viewing it in help

File: AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/query-from-gaffer.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Gaffer Simple Query Options
+
+<span style="color:red;">Note</span>: This plugin is currently not performant and is under development.
+
+This plugin is a proof of concept and does not provide useful functionality to graphs outside of its intended demonstrative use.
+
+This plugin queries data from a database that is managed by the <a href="https://gchq.github.io/gaffer-doc/v1docs/">Gaffer Framework</a>. 
+
+For this example plugin, a specific example Gaffer data set is required. Please follow the instruction on how to set up and run the <a "https://gchq.github.io/gaffer-doc/v1docs/components/example/road-traffic.html">Road Traffic data set</a>.
```
Comment (Delphinus8821) [inline AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/query-from-gaffer.md:9]: This link doesn't work

**Review by Delphinus8821 — CHANGES_REQUESTED** [review]

File: AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/adaptors-toc.xml [hunk 6]
```diff
@@ -7,11 +7,19 @@
                 <tocitem text="Extend From Pajek File" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromPajekPlugin"/>
                 <tocitem text="Extend From GraphML File" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromGraphMLPlugin"/>
                 <tocitem text="Extend From GML File" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromGMLPlugin"/>
+                <tocitem text="Extend From GDELT" target="au.gov.asd.tac.constellation.views.dataaccess.adaptors.plugins.extend.ExtendFromGDELTPlugin"/>
```
Comment (antares1470) [inline AdaptorsDataAccessPlugins/release/modules/ext/docs/AdaptorsDataAccessPlugins/src/au/gov/asd/tac/constellation/views/dataaccess/adaptors/adaptors-toc.xml:10]: Could this TOC item text be "Extend From GDELT Knowledge Graph" to match the name displayed in Data Access View and the title of the help page?

**Review by Delphinus8821 — APPROVED** [review]

**Review by antares1470 — APPROVED** [review]

**Pull request merged**